src/ensembl/production/metadata/api/adaptors/genome.py

```diff
@@ -229,12 +255,7 @@ def fetch_genomes(self, genome_id=None, genome_uuid=None, genome_tag=None, organ
             genome_select = genome_select.filter(Genome.genome_uuid == genome_uuid)
 
         if genome_tag is not None:
-            genome_select = genome_select.filter(
-                db.or_(
-                    Assembly.url_name.in_(genome_tag),
-                    Assembly.tol_id.in_(genome_tag)
-                )
-            )
+            genome_select = genome_select.filter(Genome.url_name.in_(genome_tag))
```

Comment:
Once upon a time, I was talking to Beth and I remember she told me that we want to search for `genome_tag` in both `url_name` and `tol_id`.

Since `tol_id` is attached to an assembly, not directly to the genome as a biological concept, I will need to double check with her

Nothing to change in the code for now, I just wanted to keep this note here

(cc @bethflint so that I won't forget)